### ISSUE TITLE
README: cache overview

### ISSUE THREAD

**Issue by vemv** [clojure-lsp/lein-clojure-lsp#4]
One thing that isn't clear from the README is whether the plugin needs/creates a cache.

Is it all automatic? Does it possibly reuse a cache from an IDE? Does it possibly reuse a cache from previous lein-clojure-lsp runs?

**Comment by ericdallo** [comment]
Yes, this is not on lein-clojure-lsp, but on clojure-lsp itself, when it runs, it checks if the classpath has changed to know if it should re-use the cache or not.
Even so, we only cache external deps, project analysis are not cached
